yoti-stub/src/services/YotiService.ts

```diff
@@ -1,91 +0,0 @@
-/* eslint-disable no-console */
```

Comment:
How come we deleted this? Was it just not being used?

Reply:
Yep wasn't being used anywhere